NaReguaApp/src/routes/app-cliente.routes.js

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import InicioCliente from '../paginas/inicio_cliente';
+
+const ClienteStack = createNativeStackNavigator();
+
+function AppClienteRoute() {
+  return (
+    <ClienteStack.Navigator initialRouteName="InicioCliente">
+        <ClienteStack.Screen name="InicioCliente" component={InicioCliente}/>
```

Comment:
Codacy has a fix for the issue: [Replace `··<ClienteStack.Screen·name="InicioCliente"·component={InicioCliente}` with `<ClienteStack.Screen·name="InicioCliente"·component={InicioCliente}·`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
      <ClienteStack.Screen name="InicioCliente" component={InicioCliente} />
```